src/components/projectDetail/ProjectTabs.tsx

```
@@ -218,7 +218,7 @@ const ProjectTabs = ({ projectData, isLoading, error }: ProjectTabsProps) => {
                 <button
                   key={tab}
                   ref={el => { tabRefs.current[idx] = el; }}
-                  className={`relative pb-4 text-xl font-medium ${activeTab === tab ? "text-main-color" : "text-sub-gray"}`}
+                  className={`relative pb-4 text-base md:text-xl font-medium ${activeTab === tab ? "text-main-color" : "text-sub-gray"}`}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)